File: hackathon-nlp2018-ui/src/app/model/textsearchresponse.ts
export class TextSearchResponse {

  text: string;
  pageNumber: number;
  pageSize: number;
  totalHits: number;
  results: RetrievedArticleInfo[];
  requestedUmlsCodes: string[];

}

export class RetrievedArticleInfo {

  position: number;
  identifier: string;
  titleES: string;
  titleEN: string;
  authors: string[];
  journal: string;
}

File: scielo_data_management/src/main/java/org/vicomtech/hackathon_nlp2018/scielo_data_management/model/TextSearchResponse.java
package org.vicomtech.hackathon_nlp2018.scielo_data_management.model;

import java.util.List;

import lombok.Data;

public @Data class TextSearchResponse {

	private String text;
	private int pageNumber;
	private int pageSize;
	private long totalHits;
	private List<RetrievedScieloArticleInfo>results;
	private List<String>requestedUmlsCodes;
	
}
